web/src/components/tooltips/TooltipWrapper.tsx

```diff
@@ -17,10 +18,27 @@ export default function TooltipWrapper(
   if (!tooltipContent) {
     return children;
   }
+  const [isOpen, setIsOpen] = useState(false);
+  const { isMobile } = useDimensions();
```

Comment:
Let's use our existing hook instead: `const isBiggerThanMobile = useBreakpoint('sm');`

Reply:
the sm breakpoint is a lot smaller than what we use as isMobile on the map, should it not be md? Or should we change the map as well?

Reply:
Ended up using isMobile and negated the value of the hook (was quicker and less error prone than changing reversing all the logic.

I also used `'md'` to match what it was before, but I'm open to changing this.